### ISSUE TITLE
structure defind with #pragma pack(push, 1) will not be imported

### ISSUE THREAD

**Issue by 4B5F5F4B** [joxeankoret/diaphora#134]
If you use HexRaysPyTools to define a new structure, "#pragma pack(push, 1)" will be added automatically like this
```
#pragma pack(push, 1)
struct MY_STRUCT
{
   _QWORD dummy1;
   _QWORD dummy2;
}
```

And then if you want to use diaphora_import.py to import the above structure from another idb file, some error message will output:

![error](https://user-images.githubusercontent.com/19218802/48662939-3e07fa80-eac4-11e8-97ab-5958a288da73.png)

**Comment by joxeankoret** [comment]
I think I know what is the bug. Thanks for reporting!

**Comment by joxeankoret** [comment]
Fixed with 6718a29bda7b1395d79f3eac19b2b1807285a1ce
